packages/api/prisma/schema.prisma

```diff
@@ -8,9 +8,31 @@ generator client {
 }
 
 model User {
-  id       Int    @id @default(autoincrement())
-  githubId Int    @unique
-  fullName String
-  email    String
-  image    String
+  id              Int     @id @default(autoincrement())
+  githubId        Int     @unique
+  fullName        String?
+  email           String?
```

Comment:
💭 I would consider making those properties empty strings if they are not set instead of making them nullable. It would allow us to have only one user type instead of specifying `User` and `RegisteredUser`